src/components/user-group/uc/classes.uc.js

```diff
@@ -0,0 +1,24 @@
+const { classesRepo } = require('../repo');
+
+const getClassesForStudent = async (userId) => {
+	return classesRepo.getClassesForStudent(userId);
+};
+
+const getClassesForTeacher = async (userId) => {
+	return classesRepo.getClassesForTeacher(userId);
+};
+
+const removeStudentFromClasses = async (userId, classIds) => {
+	return classesRepo.removeStudentFromClasses(userId, classIds);
+};
+
+const removeTeacherFromClasses = async (userId, classIds) => {
+	return classesRepo.removeTeacherFromClasses(userId, classIds);
+};
+
+module.exports = {
+	getClassesForStudent,
```

Comment:
removeStudentFromClasses,
removeTeacherFromClasses,

Is the combined task and should not use single. 
But i do not understand why we not delete students, teachers and substitution teachers in one step.